bootstrap/sql/postgresql/v001__create_tables.sql

```diff
@@ -39,8 +39,9 @@ CREATE TABLE IF NOT EXISTS widget (
   "type" VARCHAR(255) NOT NULL,
   "data" TEXT NOT NULL,
   "timestamp"  BIGINT,
+  "dashboardId" BIGINT NOT NULL,
   CONSTRAINT widget_uk_name UNIQUE ("name"),
-  "dashboardId"  INTEGER REFERENCES dashboard
+  FOREIGN KEY ("dashboardId") REFERENCES dashboard("id")
```

Comment:
I think foreign key constraint has to be captured in a separate script as mentioned above